Tests/MarkdownTests/Parsing/BlockDirectiveParserTests.swift

```diff
@@ -955,4 +955,31 @@ class BlockDirectiveArgumentParserTests: XCTestCase {
         """#
         XCTAssertEqual(document.debugDescription(options: .printSourceLocations), expectedDump)
     }
+    
+    func testSingleLineDirective() {
+        let source = """
+        @xx { yy }
+        @xx { yy } zz }
+        @xx { yy
```

Comment:
Was there a specific reason to drop this trailing content? Part of me thinks it should be included in the directive body.

Reply:
That's interesting. But it may need more work to achieve this.
And I think this kind of usage is very rare and is not recommended.
We should use either
```swift
@xx {
	yy
	z
}
```
or
```swift
@xx { yy z }
```

Since this PR is an additive and is an implementation-only detail.

I think this is room for future PR to support this if you think this kind of grammar should be supported.

Reply:
I agree that enabling the `@xx { yy z }` form is more important than getting everything right immediately. This looks good to go.